src/scripts/text.coffee

```diff
@@ -308,22 +311,24 @@ class ContentEdit.Text extends ContentEdit.Element
 
     _keyLeft: (ev) ->
         selection = ContentSelect.Range.query(@_domElement)
-        unless selection.get()[0] == 0 and selection.isCollapsed()
+        unless @_atStart(selection) and selection.isCollapsed()
             return
 
         # If we're at the start of the element and the selection is collapsed we
-        # should navigate to the previous text node.
+        # should navigate to the previous navigable node.
         ev.preventDefault()
+        ev.stopPropagation()
```

Comment:
This (as well as the other addition of `ev.stopPropagation()`) will cause a few tests to fail unless you add 
dummy implementation in [`src/spec/text.coffee`](https://github.com/GetmeUK/ContentEdit/blob/8176f1d02ce083b7d23ef0b6249515324fa3a8da/src/spec/text.coffee#L386). (You might also have to recompile the specs using `grunt spec`.)